app/controllers/courses/applications_open_controller.rb

```diff
@@ -1,15 +1,14 @@
 module Courses
   class ApplicationsOpenController < ApplicationController
     before_action :build_recruitment_cycle
-    before_action :build_course_params, only: :update
+    before_action :build_course_params, only: %i[update continue]
     include CourseBasicDetailConcern
 
     def update
       super
     end
 
     def continue
-      build_course_params
       super
```

Comment:
Did we have a reason why we're leaving these method shells? Just as a signal to the reader that these should be implemented somewhere?

Reply:
Yes, rubocop now gives us an error if we set a callback for something that hasn't been implemented.